docs/schema.md

```diff
@@ -740,6 +740,19 @@ this opens up the possibility for false negatives, which is why `fixed` is
 overwhelmingly preferred. An [example](#last_affected-vs-fixed-example) is available to
 illustrate the difference.
 
+The `fixed` and `limit` events are closely related (refer to the [Limit events
+example](#Limit-events) for further details) and involve a similar trade-off.
+Consider a branch forked from `main` after a vulnerability introduction
```

Comment:
This is a lot of text to describe a situation that would likely be difficult to visualise for readers unfamiliar with these concepts. We should instead lean on the example linked to describe this. 

Would you agree with something a bit more concise, along the lines of:

The `fixed` and `limit` events are closely related and involve a similar trade-off. See the [example](#last_affected-vs-fixed-example) for an example of how they differ. `events` arrays with `fixed` events must include all other cherrypicked fix commits in all branches as separated `fixed` events to avoid false positive matches in other branches. Unlike `fixed` events, `limit` events restrict the set of vulnerable commits to ones that are reachable from the `limit`, which may result in false negatives. Where possible, it's strongly recommended to use `fixed` over `limit`.

Reply:
Yes, sure. I've updated the PR to reflect the more concise version, with minor copy editing changes.
I've also included in it the fix for the other issue you mentioned (highlighting this is relevant for GIT ranges only).